### PR TITLE
Rename some functions to something easier understand

### LINKED ISSUE #121: Rename helper functions like `getS` to nicer names
random suggestions from the top of my head:

- `getS` → `stoichiometry_matrix`
- `getUB/LB` → `flux_bounds` (returns both, I can't see how anyone would want just lower or upper bounds)
- `getOC` → `flux_objective`

cc @giordano

### PR DESCRIPTION
**Pull request by giordano**
* `getS` -> `stoichiometry_matrix`
* `getLBs`, `getUBs` -> `flux_bounds`
* `getOCs` -> `flux_objective`

Fix #121.

### CODE REVIEW

File: test/ecoli_flux.jl [hunk 1]
```diff
@@ -41,17 +41,16 @@ end
...
+    ocs = flux_objective(mdl)
 
     @test length(ocs) == length(mets)
     @test ocs[40] == 1.0
     deleteat!(ocs, 40)
     @test all(ocs .== 0.0)
 
-    @test length(getLBs(mdl)) == length(rxns)
-    @test length(getUBs(mdl)) == length(rxns)
+    @test length(flux_bounds(mdl)[2]) == length(rxns)
```
Comment (exaexa) [inline test/ecoli_flux.jl:52]: there should be `[1]` here right?

**Review by exaexa — CHANGES_REQUESTED** [review]
LGTM, except the [1] vs [2] index.

File: test/ecoli_flux.jl (hunk 1 above)
Comment (giordano) [reply test/ecoli_flux.jl]: Ugh, yes, thanks!  Fixed

**Comment by exaexa** [comment]
oh noes I trusted the github "resolve conflicts" button :D

**Review by exaexa — APPROVED** [review]

**Pull request merged**